src/hooks/useUrlId.ts

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react';
 import { getQueryParam } from 'uno-js';
 
-export const useUrlId = (idName: string, window: any): [number, (event: any) => void, () => void, () => void] => {
+export const useUrlId = (
+    idName: string,
+    window: any,
+): [number | string, (event: any) => void, () => void, () => void] => {
```

Comment:
Maybe you can check what are you using internally:

https://github.com/unosquare/uno-js/blob/master/src/getQueryParam.ts#L1